lib/brain.js

```diff
@@ -1883,6 +1949,12 @@ Brain.prototype.isTierCompliant = function isTierCompliant (tier) {
     case 'block':
     case 'suspend':
       return false
+    case 'custom':
+      const customInfoRequestId = tier
+      const customerData = _.find(['info_request_id', customInfoRequestId])(customer.customInfoRequestData)
+      if (!customerData || customerData.approved === false) return false
+      // when approved is null then data was already added but not manually approved yet by the operator
+      return true
```

Comment:
My logic might be faulty, but can't this be simplified to:

```suggestion
      return !!customerData && customerData.approved !== false
```